src/utils.ts

```diff
@@ -14,3 +14,13 @@ function getLastActiveRowIndex(column: string): number {
     .getRow();
   return index;
 }
+
+function opcodeAssertions(instruction: decodedInstruction, operandsValue: operands): void{
+  switch (instruction.Opcode){
+    case Opcodes.AssertEq:
+      if (Number(operandsValue.dst) !== Number(operandsValue.res)){
+        throw new AssertEqError();
+      }
+      break;
+  }
+}
```

Comment:
configure your IDE to automatically add new lines at the end of files, in VS Code it is

```
  "files.insertFinalNewline": true,
  "files.trimFinalNewlines": true,
  "files.trimTrailingWhitespace": true,
```